system-int-test/test/cmr/system_int_test/ingest/variable_ingest_test.clj

```diff
@@ -213,15 +202,31 @@
         {:keys [status]} (ingest/ingest-concept
                           concept
                           (variable-util/token-opts token))]
-    (index/wait-until-indexed)
     (is (= 201 status))))
 
+(deftest veriable-ingest-schema-validation-test
```

Comment:
variable typo

Reply:
Wondering when can I not make a typo. :)